src/models/modules/bricks/droppath.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+def drop_path(x, drop_prob: float = 0., training: bool = False, scale_by_keep: bool = True):
```

Comment:
Depending on whether Swin gets merged first, you should take one of the two implementations